dev/repo/reset.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
I think I remember why we have this in this repo, it is for testing out github actions

Reply:
It was indeed originally for that and now formalized to do a little more (see [WIP docs](https://github.com/appoptics/appoptics-bindings-node/tree/AO-20192-Update-Documentation-no-build#dev-repo)). 

Given that the repo is the *"production environment"* for GitHub Actions, makes sense to have *"development environment"* somewhere. I tried [Act](https://github.com/nektos/act) to run things locally, but it is lacking, so for now I guess dev repos ae the way to go for complex GitHub Action flows...